src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -465,6 +465,9 @@ actionRegistry.registerWorkbenchAction(new SyncActionDescriptor(RenameTerminalAc
 actionRegistry.registerWorkbenchAction(new SyncActionDescriptor(FocusTerminalFindWidgetAction, FocusTerminalFindWidgetAction.ID, FocusTerminalFindWidgetAction.LABEL, {
 	primary: KeyMod.CtrlCmd | KeyCode.KEY_F
 }, KEYBINDING_CONTEXT_TERMINAL_FOCUS), 'Terminal: Focus Find Widget', category);
+actionRegistry.registerWorkbenchAction(new SyncActionDescriptor(FocusTerminalFindWidgetAction, FocusTerminalFindWidgetAction.ID, FocusTerminalFindWidgetAction.LABEL, {
+	primary: KeyMod.CtrlCmd | KeyCode.KEY_F
+}, KEYBINDING_CONTEXT_TERMINAL_FIND_WIDGET_FOCUSED), 'Terminal: Focus Find Widget', category);
```

Comment:
Was `KEYBINDING_CONTEXT_TERMINAL_FOCUS` not true when the find widget is focused? This will cause problems for users who want to configure custom keybindings with `terminalFocus`

Reply:
`KEYBINDING_CONTEXT_TERMINAL_FOCUS` does not appear to be true when the find widget is focused. What kinds of problems will this cause for custom keybindings?